libs/logging/src/interfaces.ts

```diff
@@ -0,0 +1,22 @@
+export type LogContext = {
+  // fingerprint is a Rollbar concept that helps Rollbar group error occurrences together
+  fingerprint?: string;
+  [key: string]: unknown;
+};
+
+/**
+ * Logger port
+ * Logs messages at different levels
+ */
+export interface ILogger {
+  trace(msg: string, context?: LogContext): void;
+  debug(msg: string, context?: LogContext): void;
+  info(msg: string, context?: LogContext): void;
+  warn(msg: string, context?: LogContext): void;
```

Comment:
you can argue that warn is closer to errors than infos, suggesting that should follow the error interface